src/General/Typography/Paragraph.test.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import Paragraph from './Paragraph';
+import { PrimaryColor } from '../../Style/Colors';
+
+import '@testing-library/jest-dom/extend-expect';
+import { render } from '@testing-library/react';
+
+import {
+  PARAGRAPH_FONT_SIZES,
+  PARAGRAPH_VARIANTS,
+  paragraphType,
+} from './ParagraphStyles';
+
+const paragraphText =
+  'As the first sign in the zodiac, the presence of Aries always marks the beginning of something energetic and turbulent. They are continuously looking for dynamic, speed and competition, always being the first in everything - from work to social gatherings.';
+
+describe('displays the correct text and font-size for each variant:', () => {
+  const variants: paragraphType[] = Object.values(PARAGRAPH_VARIANTS);
```

Comment:
`const variants = Object.values(PARAGRAPH_VARIANTS) as paragraphType[];`

Reply:
@franciscrispin do you have an error `Property 'values' does not exist on type 'ObjectConstructor'` for `Object.values`?
According to https://stackoverflow.com/questions/52304422/property-values-does-not-exist-on-type-objectconstructor, I think we can simply add `es2017.object` to `compilerOptions` `lib` array to resolve it.

Reply:
@rosslo Yeap that error is fixed in the `v4` branch already. I just need to merge it in and it should be fine.